appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/util/ModelUtils.java

```diff
@@ -569,37 +546,39 @@ else if (newValue instanceof Constructible) {
     }
 
     public static String getResourcePath(GenericDeclaration declaration, Map<String, Set<Class<?>>> resourceMapping) {
```

Comment:
FYI: split this in a method for `Class` and `Method` each which now use `AnnotationInfo` to check annotation so "inherited" annotations are taken into account.